miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/MiniAppAnalytics.kt

```diff
@@ -43,5 +44,5 @@ internal class MiniAppAnalytics(val rasProjectId: String) {
         params["cp"] = cp
 
         RatTracker.event(eType.value, params).track()
-    }
+    } catch (e: Exception) {}
```

Comment:
Maybe we should add a `Log.e` here? Seems unlikely that  this block will be hit, but it could be tricky to track done if there's ever an error here.

Reply:
Updated.